### PR TITLE
First lab

### PR DESCRIPTION
**Pull request by cseydlitz**
@anthonybotello Please Review. 
I am skeptical about my answers for Problem 3 parts C, D, and E
Also if you have any pointers on how to better explain parts D and E of the final question, I am open to suggestions.

### CODE REVIEW

**Comment by anthonybotello** [comment]
I'm not sure why you're importing everything in each individual cell. There isn't anything necessarily wrong with it, clearly, as you were able to complete the assignment. However, stylistically, I think it's more concise to have all of your imports at the beginning of the notebook.

My only real issue is with your explanation for part 1.D. You haven't addressed why you would get a negative number or what it would mean experimentally.

**Comment by cseydlitz** [comment]
@anthonybotello Thank you for the feedback, changes made as suggested. Good to know about the imports; I didn't know you could just have one cell for all imports. 
Final workbook is up.

@oliviatessa Please review

File: LAB1.html [hunk 1]
```diff
@@ -0,0 +1,13543 @@
...
+fieldset[disabled] .btn-info.focus {
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+.btn-info .badge {
+  color: #5bc0de;
+  background-color: #fff;
+}
+.btn-warning {
+  co
```
Comment (oliviatessa) [inline LAB1.html:13144]: In future labs, I don't think it is necessary to explain the methods you create in python. I'd rather you spend more time discussing the statistical concepts: probability, sigma, etcs.

Comment (oliviatessa) [inline LAB1.html:13144]: Explain what the normal distribution is, what parameters it depends on, and the meaning of sigma.

Comment (oliviatessa) [inline LAB1.html:13204]: Make sure you interpret your results and discuss whether they make sense.

Comment (oliviatessa) [inline LAB1.html:13279]: Don't forget important details in your statistical question such as the nature of your background distribution.

Comment (oliviatessa) [inline LAB1.html:13296]: This wouldn't be the sigma value that answers your statistical question. Sigma is meaningless for the Rayleigh distribution. Sigma is the value for the standard normal distribution that corresponds to your calculated probability.

Comment (oliviatessa) [inline LAB1.html:13457]: Interesting, why is it appropriate to ask this question? Think about the difference between continuous and discrete distributions.

Comment (oliviatessa) [inline LAB1.html:13523]: Can the mean of a Poisson distribution be non integer? What constraints does a discrete distribution put on sigma and probabilities?

**Comment by oliviatessa** [comment]
Overall, I think your writeup could use more writing, particularly for the basic concepts explored at the beginning of the lab. Also, I think you need to revisit the concept of "sigma" as a probability. 

Some of your plots could be reformatted to be more clear. In particular, your Poisson plots were hard to follow.

**Pull request closed without merging**